modules/pull_request_builder/buildspecs/central_audit_terraform_plan.yaml

```diff
@@ -10,5 +10,4 @@ phases:
     commands:
       - make lambda-ci
       - make fmt-check
-      - ./scripts/run.sh plan development
-      - ./scripts/run.sh plan production
+      - ./scripts/run.sh plan development && ./scripts/run.sh plan production
```

Comment:
I think if anything I'd like a plan for both envs plan to run in parallel. Speeding up the PR process can really improve interaction with this repo. 

It'd be interesting if we could use something like the build-matrix functionality to do this. Maybe set an ${ENVIRONMENT} ENV for each build. It'd certainly make the PR builder faster.
https://docs.aws.amazon.com/codebuild/latest/userguide/batch-build-buildspec.html#build-spec.batch.build-list

Maybe one to chat about as a team